authentication_executor/authenticator.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, bucket_uploader: BucketUploaderABC, url_signer: URLSignerABC,
         self.api_client = api_client
```

Comment:
All changes in this file are from `TRA-2938_oauth_support` branch

Reply:
I think you have mistake and it is frmo another branch because you wrote the name of this branch../
I think we don't need those changes - make code conditionally when there is no real reason (you can pass mock from outside - there is example in agent)